data/query.go

```diff
@@ -0,0 +1,97 @@
+package data
+
+import (
+	"context"
+	"net/url"
+	"strconv"
+
+	"github.com/ONSdigital/log.go/log"
+)
+
+const (
+	// DefaultLimit - default values for limit query
+	DefaultLimit = 10
+	// DefaultSort - default values for sort query
+	DefaultSort = "relevance"
+	// DefaultPage - default values for page query
+	DefaultPage = 1
+)
+
+// GetLimitOptions returns all available limit options for search
+func GetLimitOptions() []int {
+	return []int{10, 25, 50}
+}
+
+// UpdateQueryWithOffset - removes page key and adds offset key to query to be then passed to dp-search-query
+func updateQueryWithOffset(ctx context.Context, query url.Values) url.Values {
+
+	page, err := strconv.Atoi(query.Get("page"))
+	if err != nil {
+		log.Event(ctx, "unable to convert search page to int - set to default "+strconv.Itoa(DefaultPage), log.INFO)
+		page = DefaultPage
+	}
+	limit, err := strconv.Atoi(query.Get("limit"))
+	if err != nil {
+		log.Event(ctx, "unable to convert search limit to int - set to default "+strconv.Itoa(DefaultLimit), log.INFO)
```

Comment:
Same here.

Reply:
Same here too ;)

Reply:
I have added a string const for those values and hence it is more readable